include/picongpu/fields/currentDeposition/VillaBune/CurrentVillaBune.hpp

```diff
@@ -249,10 +249,10 @@ namespace currentSolver
         )
         {
 
-            if (math::float2int_rd(oldPos.y()) != math::float2int_rd(newPos.y()))
+        if (pmacc::math::float2int_rd(oldPos.y()) != pmacc::math::float2int_rd(newPos.y()))
             {
                 const float3_X interPos = intersectYPlane(oldPos, newPos,
-                                                          math::max(math::float2int_rd(oldPos.y()), math::float2int_rd(newPos.y())));
+                                                      math::max(pmacc::math::float2int_rd(oldPos.y()), pmacc::math::float2int_rd(newPos.y())));
```

Comment:
E.g. here it also looks inconsistent now with mix of `pmacc::` and no `pmacc::` for similarly twin functions.

Reply:
`math` is pointing to `cupla::math`. PIConGPU is used by physicists, I tried to avoid using `cupla::math` or `cupla::max` in PIConGPU.